tmt/trying.py

```diff
@@ -680,28 +720,29 @@ def go(self) -> None:
             self.action_verbose(plan)
 
         # Choose the initial action
-        action = Action.START_ASK
+        action = find_action("start_ask")
         if self.opt("login"):
-            action = Action.START_LOGIN
+            action = find_action("start_login")
         elif self.opt("ask"):
-            pass  # already START_ASK
+            pass  # already start_ask
         elif self.tests:
-            action = Action.START_TEST
+            action = find_action("start_test")
 
         # Loop over the actions
         try:
             while True:
                 # Choose the verbose and debug level
-                if action in [Action.VERBOSE, Action.DEBUG]:
-                    getattr(self, f"prompt_{action.action}")()
+                action_name = action.full_name
+                if action_name in ["verbose", "debug"]:
+                    getattr(self, f"prompt_{action_name}")()
```

Comment:
IIUIC, `prompt_{action_name}` code should be moved to their respective actions. What happens here: `prompt_*` is called first for `debug` and `verbose`, and then the corresponding action is invoked. By moving their "prompt" half into their action handlers we can drop this list of "actions that have special class method detached from their actions".

Reply:
Handled via ActionInfo now